tests/ceph_installer/test_ansible.py

```diff
@@ -146,6 +147,18 @@ def run(ceph_cluster, **kw):
 
     LOG.info("Ceph ansible version " + ceph_installer.get_installed_ceph_versions())
 
+    # Check if ansilbe.log file has any logs initially
+    if custom_log:
+        ansible_log_file, _ = ceph_installer.exec_command(
+            cmd=f"grep log_path {ansible_dir}/ansible.cfg | cut -d ' ' | tr -d '\n\r'"
```

Comment:
I believe we would need to escape the backlash... this is the output from the log
```
2022-08-26 08:49:56,514 - INFO - cephci.test_ansible:110 - Running command du  | cut -f1 | tr -d '

' on 10.0.208.48 timeout 600
```

Reply:
@psathyan, we'll need a backslash as the value it is returning is with a new line, hence I've used `tr -d '\n\r'`

Reply:
@adityaramteke As @psathyan mentioned, the backslash has to be escaped. 

Please check the output of the sample code without escaping the backslash

cmd = f"du {ansible_log_file} | cut -f1 | tr -d '\n\r'" 
=> du pranav/temp | cut -f1 | tr -d '
'

where as with backslash escaped, 
cmd = f"du {ansible_log_file} | cut -f1 | tr -d '\\n\\r'"
=> du pranav/temp | cut -f1 | tr -d '\n\r'

which is the expected command, if I'm not wrong.

Reply:
@pranavprakash20, I'm still getting the same output for both with and without escaping the backslash! 

``` 
cmd = f"du {ansible_log_file} | cut -f1 | tr -d '\n\r'"
du aramteke/temp | cut -f1 | tr -d '
'

cmd = f"du {ansible_log_file} | cut -f1 | tr -d '\n\r'"
du aramteke/temp | cut -f1 | tr -d '
'
```
What's the actual difference?

Reply:
Got the requested changes now, 

```
cmd = f"du {ansible_log_file} | cut -f1 | tr -d '\\n\\r'"
```
Thanks, @pranavprakash20 and @psathyan! 
I'm updating the changes and will post the script result!